.github/workflows/bump.yml

```diff
@@ -0,0 +1,187 @@
+        run: |
+          OVERLAYS=""
+
+          # Function to add overlay path with comma if needed
+          add_overlay() {
+            local overlay_path="$1"
+            if [ -f "$overlay_path" ]; then
+              if [ -n "$OVERLAYS" ]; then
+                OVERLAYS="$OVERLAYS,$overlay_path"
+              else
+                OVERLAYS="$overlay_path"
+              fi
+            fi
+          }
+
+          # Check for doc-specific overlays (list each overlay file individually)
+          if [ -d "${{ matrix.doc_id }}/overlays" ]; then
+            find "${{ matrix.doc_id }}/overlays" -name "*.yaml" -type f | sort | while read overlay_file; do
+              add_overlay "$overlay_file"
+            done
+          fi
+
+          # Check for shared overlays - only apply to cloud APIs
+          if [[ "${{ matrix.doc_id }}" == "cloud-"* ]] && [ -d "shared/overlays" ]; then
+            find "shared/overlays" -name "*.yaml" -type f | sort | while read overlay_file; do
+              add_overlay "$overlay_file"
+            done
+          fi
+
+          echo "overlay_paths=$OVERLAYS" >> $GITHUB_OUTPUT
+          echo "Using overlays: $OVERLAYS"
+
+      - name: Deploy API documentation
+        uses: bump-sh/github-action@v1
+        with:
+          hub: redpanda
+          doc: ${{ matrix.doc_id }}
+          token: ${{secrets.BUMP_TOKEN}}
+          file: ${{ steps.format.outputs.file_path }}
+          overlay: ${{ steps.overlays.outputs.overlay_paths }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  api-diff:
+    if: ${{ github.event_name == 'pull_request' && fromJson(needs.determine-doc-ids.outputs.matrix).doc_id[0] != null }}
+    needs: determine-doc-ids
+    name: Check API diff on Bump.sh
+    runs-on: ubuntu-latest
+    strategy:
+      matrix: ${{fromJson(needs.determine-doc-ids.outputs.matrix)}}
+      fail-fast: false
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Determine file format
+        id: format
+        run: |
+          if [ -f "${{ matrix.doc_id }}/${{ matrix.doc_id }}.yaml" ]; then
+            echo "file_path=${{ matrix.doc_id }}/${{ matrix.doc_id }}.yaml" >> $GITHUB_OUTPUT
+          elif [ -f "${{ matrix.doc_id }}/${{ matrix.doc_id }}.json" ]; then
+            echo "file_path=${{ matrix.doc_id }}/${{ matrix.doc_id }}.json" >> $GITHUB_OUTPUT
+          elif [ -f "${{ matrix.doc_id }}/${{ matrix.doc_id }}-api.json" ]; then
+            echo "file_path=${{ matrix.doc_id }}/${{ matrix.doc_id }}-api.json" >> $GITHUB_OUTPUT
+          else
+            echo "No API definition file found for ${{ matrix.doc_id }}"
+            exit 1
+          fi
+
+      - name: Comment pull request with API diff
+        uses: bump-sh/github-action@v1
+        with:
+          hub: redpanda
+          doc: ${{ matrix.doc_id }}
+          token: ${{secrets.BUMP_TOKEN}}
+          file: ${{ steps.format.outputs.file_path }}
+          command: diff
```

Comment:
I'm curious about the diff comment that was added to the PR. I don't see changes to the API definitions (just the filenames if I'm understanding correctly) in the PR but it's identified a few changes to properties. Is this expected?

Reply:
It's diffing the content in this PR against what is currently uploaded on Bump.